internal/config/server_configuration_test.go

```diff
@@ -52,8 +59,10 @@ server: "should be a table, not a string"`
 
 	config, err := config.GetServerConfiguration(filename)
 	assert.Error(t, err)
-	assert.Equal(t, 0, config.Server.Port)
-	assert.Equal(t, "", config.Database.Url)
+	assert.Equal(t, 0, config.Server.GSPort)
```

Comment:
NIT: This can probably be `assert.Equal(t, ServerConfig{}, config)`

Reply:
Fixed in 52268e8